api/openapi_server/controllers/auth_controller.py

```diff
@@ -13,6 +13,7 @@
 from sqlalchemy.exc import IntegrityError
 from sqlalchemy import select
 
+from botocore.exceptions import ClientError
```

Comment:
using botocore errors for cleaner code?

Reply:
Yes, because the old approach was error prone.

```python
    except Exception as e:
        code = e.response['Error']['Code']
```

We are catching all exceptions and then attempting to access a `response` property. `Exception` does not have a `response` property, so if an unrelated exception was thrown then `e.response` would cause another exception to occur.